contrib/token-server/main.go

```diff
@@ -120,9 +128,52 @@ func handleError(ctx context.Context, err error, w http.ResponseWriter) {
 	context.GetResponseLogger(ctx).Info("application error")
 }
 
+var refreshCharacters = []rune("0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+const refreshTokenLength = 15
+
+func newRefreshToken() string {
+	s := make([]rune, refreshTokenLength)
+	for i := range s {
+		s[i] = refreshCharacters[rand.Intn(len(refreshCharacters))]
```

Comment:
doesn't matter to me since this is intended to be used as tests. Seeding with time could ensure that tests are not always being run against the same refresh tokens though.